src/components/EnergyCalendar.tsx

```diff
@@ -0,0 +1,193 @@
+              </div>
+              <div className="statistical-content">
+                <div className="statistical-content-title">上月非工作日</div>
+                <div className="statistical-content-title__sub">平均能耗</div>
+                <div className="statistical-content-value">{todayEnergy.toFixed(1)}</div>
+                <div className="statistical-content-unit">
+                  MWh<span>e</span>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </Container>
+  );
+};
+
+const Container = styled.div`
+  width: 100%;
+  height: 100%;
+  padding: 20px;
+  display: flex;
+  border-radius: 20px;
+  background-color: #ffffff;
+  flex-direction: column;
+  .header {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    font-size: 24px;
+    font-weight: 700;
+    line-height: 24px;
+  }
+  .body {
+    flex: 1;
+    display: flex;
+    margin-top: 16px;
+    .calendar-custom {
+      width: 430px;
+      border: 0px;
+      .react-calendar__tile {
+        padding: 1.2em 1em;
+        font-size: 18px;
+      }
+      .react-calendar__month-view__weekdays {
+        height: 0px;
+        .react-calendar__month-view__weekdays__weekday {
+          padding: 0em;
+        }
+      }
+    }
+  }
+  .data-body {
+    flex: 1;
+    display: flex;
+    margin-top: 16px;
+    .circle {
+      position: relative;
+      div {
+        width: 144px;
+        height: 144px;
+        border-radius: 50%;
+        position: absolute;
+      }
+      .circle-notch {
+        z-index: 1;
+        background-color: #ffffff;
+        left: 50px;
+        top: -55px;
+      }
+      .circle-base {
+        border: 12px solid #00c147;
+        left: 50px;
+        top: 25px;
+        box-sizing: border-box;
+        div {
+          z-index: 2;
+          text-align: center;
+          font-family: HarmonyOS Sans SC;
+          position: absolute;
+          width: 100%;
+          color: #999999;
+          font-style: normal;
+          font-size: 14px;
+        }
+        .circle-title {
+          top: 12px;
+        }
+        .circle-value {
+          top: 32px;
+          line-height: 48px;
+          font-weight: bold;
+          font-size: 36px;
+          color: #333333;
+        }
+        .circle-unit {
+          top: 82px;
+          span {
+            vertical-align: sub;
+          }
+        }
+      }
+    }
+    .average-statistical-content {
+      position: relative;
+      .statistical-content-body {
+        position: absolute;
+        width: 246px;
+        height: 160px;
+        bottom: 0px;
+        .statistical-content {
+          width: 50%;
+          height: 100%;
+          float: left;
+          position: relative;
+          div {
+            text-align: center;
+            position: absolute;
+            width: 100%;
+            color: #999999;
+            font-size: 14px;
+          }
+          .statistical-content-title {
```

Comment:
```suggestion
          &-title {
```
在`styled-component`的`css`中你可以使用`&`来指代上层组件以避免重复输入`.statistical-content`。可以参考[以下文档](https://styled-components.com/docs/basics#pseudoelements-pseudoselectors-and-nesting)。